packages/@aws-cdk/aws-codepipeline/README.md

```diff
@@ -339,3 +339,14 @@ myPipeline.onStateChange('MyPipelineStateChange', target);
 myStage.onStateChange('MyStageStateChange', target);
 myAction.onStateChange('MyActionStateChange', target);
 ```
+
+## CodeStar Notifications
+
+Pipelines can be used as a source for notification.
+To define CodeStar Notification rules for build projects, use one of the `notifyOnXxx` methods, it very similar to Amazon CloudWatch event `onXxx` method.
+
+For example:
+
+```ts
+pipeline.notifyOnPipelineStateChange('NotifyOnPipelineStateChange', { target });
```

Comment:
Show a target you can use with these, like you did for CodeBuild.

Reply:
Already adjusted.